modules/gitops-release-notes-1-12-0.adoc

```diff
@@ -0,0 +1,142 @@
+:_mod-docs-content-type: REFERENCE
+
+[id="gitops-release-notes-1-12-0_{context}"]
+= Release notes for {gitops-title} 1.12.0
+
+{gitops-title} 1.12.0 is now available on {OCP} 4.12, 4.13, 4.14, and 4.15.
+
+[id="errata-updates-1-12.0_{context}"]
+== Errata updates
+
+[id="gitops-1-12-0-security-update-advisory_{context}"]
+=== RHSA-xxxx:nnnn - {gitops-title} 1.12.0 security update advisory
```

Comment:
@Dhruv-Soni11 Errata info for now to get updated in the RN:

- RHSA-2024:127897

TY!

Reply:
Fixed.